master/pkg/schemas/expconf/searcher_config.go

```diff
@@ -19,6 +19,8 @@ type SearcherConfigV0 struct {
 	RawAdaptiveASHAConfig *AdaptiveASHAConfigV0 `union:"name,adaptive_asha" json:"-"`
 	RawCustomConfig       *CustomConfigV0       `union:"name,custom" json:"-"`
 
+	// TODO(DET-8577): There should not be a need to parse EOL searchers if we get rid of parsing
```

Comment:
objectively seems like we could want to parse EOL searcher info for ever, for informational reasons. not parsing searcher on completed experiments seems like an unnecessary limitation.

Reply:
Fetching the name?  Sure.

But what else would we want to parse?  PBT appears to have been stricken from the record, nobody seems to have complained.

Reply:
true. if anything, we could just display the old conf in a blob and call it a day.